docs/paas-plus/dev/platform-as-a-service-plus.md

```diff
@@ -0,0 +1,161 @@
+
+
+**Step 4. Connect your repository***
+*Feature only available for Super Users.
+
+Connect your repository to a project, before running the pipelines. Open the“Code” section, click on the kebab menu and select "Switch repository or Git provider”. 
+
+![gif1](https://user-images.githubusercontent.com/83701393/176354903-a9d0a669-af3f-449d-bde7-cfc696e96569.gif)
+
+
+Select your git provider & add a new integration. Authorization is possible via OAuth or Token. After connecting, choose “Clone repository from scratch”.
+
+![gif2](https://user-images.githubusercontent.com/83701393/176354926-c486ddfc-25e2-4ae6-b32d-d0dad14d4aee.gif)
+
+
+
+**Step 5. Push & have fun!**
+Main pipeline in each project is triggered when you commit & push from your connected repository. More information on Pipelines is provided below.       
+<img width="446" alt="Image 5 6" src="https://user-images.githubusercontent.com/83701393/176354944-9d097014-9f1e-4ec7-9788-287f447fde4a.png">
+
+
+
+
+## What pipelines are automatically included in your workspace?
+
+**###Main pipeline inside environment-related projects**
+The main pipeline inside a project is named as follows: 
+[“***” + ”environment_name” + ”Pipeline” + ”***”]. 
+<img width="446" alt="Image 5 6" src="https://user-images.githubusercontent.com/83701393/176354996-677d9051-1a9f-4527-85d0-9c3ee75404e0.png">
+
+
+Examples of main pipelines names:  “*** Dev Pipeline ***” or “*** Prod Pipeline ***”
+
+
+The main pipeline is triggered when you push from your connected repository, and it automatically triggers the remaining pipelines, according to the following actions:
+<img width="186" alt="Image7" src="https://user-images.githubusercontent.com/83701393/176355028-49e4b72e-19d0-49f1-9216-6ff909da7f26.png">
+
+
+####What happens in each step of the pipeline?
+Please note, the sub-pipelines (i.e. Build, Quality Gate and Deployment) **should not be run independently **and would lead to deployment failure.
+
+**1. Build**
+- Builds Docker images
+- Pushes the image to ECR
+
+**2. Quality Gate**
+- Provides automated code quality feedback, to ensure your [project is always upgradable] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html). 
+- Checks for compliance of Spryker Core and project specific code with our customization best practices. These are relevant for BE customizations that may cause incompatibility with minor releases or patches.
+- For non-compliant code, the Quality Gate displays an issue report. Additional guidelines for further development improvements are also available in [Spryker documentation portal] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.html).
+- For an effortless & safer upgrade experience, Spryker recommends to only deploy code that passes the Quality Gate assessment.
```

Comment:
This does not look like a step of the pipeline.

Reply:
True. Is it possible to add it as a "warning" or "note" sort of section? something like this:
![Screenshot - 2022-07-04T112758 224](https://user-images.githubusercontent.com/83701393/177126017-7619d9cf-ef10-448d-b6cd-bf3f43995fc2.png)
?